File: Angular-Frontend/src/app/models/product.ts
export class Product{
    id: number;
    name: string;
    description: string;
    vender: string;
    stock: number;
    purchaseRate: number;
    sellingRate: number;
    constructor( id,name,description,vender,stock,purchaseRate, sellingRate) {
        this.id = id;
        this.name = name;
        this.description = description;
        this.vender = vender;
        this.stock = stock;
        this.purchaseRate = purchaseRate;
        this.sellingRate = sellingRate;
        
    }
}

File: AspNetCore_Api/Models/Product.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace AspNetCore_Api.Models
{
    public class Product
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Vender { get; set; }
        public int Stock { get; set; }
        public decimal PurchaseRate { get; set; }
        public decimal SellingRate { get; set; }
    }
}
